spec/models/course_spec.rb

```diff
@@ -578,10 +578,10 @@
 
             let(:accrediting_provider_enrichments) { [accrediting_provider_enrichment] }
             let(:provider_enrichment) do
-              build(:provider_enrichment,
-                    :published,
-                    last_published_at: 1.day.ago,
-                    accrediting_provider_enrichments: accrediting_provider_enrichments)
+              create(:provider_enrichment,
```

Comment:
This is going to create a `provider` since 1) it's not used here, and 2) it's being created.

Reply:
Done. Thanks for the feedback.